docs/guides/schema.md

```diff
@@ -78,8 +78,8 @@ type User {
 }
 ```
 
-> **Note:** Only a subset of all the generated types/mutations/queries are shown
-> here. To see a more complete example [follow the Quick Start](/docs/quick-start/README.md)
+> **Note:** Only a subset of the generated types/mutations/queries are shown here.
+> For more details, see the [GraphQL introduction](https://www.keystonejs.com/guides/intro-to-graphql) page.
```

Comment:
For internal links please use paths, which are resolved by Gatsby for the docs site, and still work when browsing the monorepo on GitHub e.g.

```suggestion
> For more details, see the [GraphQL introduction](/docs/guides/intro-to-graphql.md) page.
```